cassdegrees/templates/managecourses.html

```diff
@@ -0,0 +1,115 @@
+{% extends "base.html" %}
+
+{% block page-title %}Manage Courses{% endblock %}
+
+{% block subtitle %}Add, Modify or Delete Courses{% endblock %}
+
+{% block content %}
+    <form class="anuform" action="/manage_courses/" method="post">
+        {% csrf_token %}
+        <fieldset>
+            <legend>Add Course</legend>
+
+            <!-- As HTML forms only support get or post methods,
+            add this hidden input for consistency with the rest of the forms -->
+            <input type="hidden" name="_method" value="post">
+
+            <p>
+                <label for="id">Course Name:</label>
+                <input class="text" id="text" type="text" name="name" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="id">Course Code:</label>
+                <input class="text" id="text" type="text" name="code" minlength="8" maxlength="8" size="8" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="text">Year:</label>
+                <!-- static min max values are only temporary -->
+                <input class="text" id="number" type="number" name="year" min="1990" max="2100" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="text">Semester(s):</label>
+                <select name="semester[]" size="2" multiple>
+                    <option value="semester1">Semester 1</option>
+                    <option value="semester2">Semester 2</option>
+                </select>
+            </p>
+
+             <p>
+                <label for="text">Units:</label>
+                 <!-- max value is chosen arbitrarily -->
+                <input class="text" id="number" type="number" name="units" min="6" max="48" step="6" aria-required="true" required>
```

Comment:
```suggestion
                <input class="text tfull" id="number" type="number" name="units" min="0" step="6" aria-required="true" required>
```

Courses can be worth 0 units, so this should be reflected here.
Stretch to end of form and removed max units for a course (Realistically it should only be 12-24 max but why restrict them?)

Reply:
my thought process was so it leaves the user less of a chance to create a mistake when creating a course, hence adding max for both year and units.